### PR TITLE
Adding a default placeholder image on the board.

### PR DESCRIPTION
**Pull request by andrewhao**
screen shot 2015-06-11 at 2 53 27 pm

Added a default image with Treyce.

### CODE REVIEW

**Comment by crsven** [comment]
:+1:

**Comment by andrewhao** [comment]
I'm working on a simple spec...

File: lib/models/board.coffee [hunk 1]
```diff
@@ -98,6 +98,9 @@ BoardSchema.methods =
     up = ( if up.getTime() > card.updated.getTime() then up else card.updated ) for card in @cards()
     up
 
+  screenshotUrlWithDefault: ->
```
Comment (andrewhao) [inline lib/models/board.coffee]: Maybe it's almost time to introduce a presenter...

File: Cakefile [hunk 2]
```diff
@@ -40,8 +40,7 @@ task 'spec:controllers', 'Run all specs in spec/lib/controllers', ->
   execCmd 'NODE_ENV=test ./node_modules/.bin/jasmine-node --captureExceptions --coffee spec/lib/controllers'
 
 task 'spec', 'Run all client and server specs', ->
-  invoke 'spec:client'
-  invoke 'spec:server'
+  execCmd './node_modules/.bin/cake spec:client && ./node_modules/.bin/cake spec:server'
```
Comment (andrewhao) [inline Cakefile]: The order of the runs here is not guaranteed, nor is the correctness of the return value.

> Cake tasks are a minimal way to expose your CoffeeScript functions to the command line, so don't expect any fanciness built-in. If you need dependencies, or async callbacks, it's best to put them in your code itself — not the cake task.

http://coffeescript.org/

Comment (crsven) [inline Cakefile]: Ah, interesting. We tried adding `invoke`s for the other spec tasks and it went to hell. Would adding them to this execCmd list work? Seems like having them all run from `cake spec` would be ideal.

Comment (andrewhao) [inline Cakefile]: I think `cake spec` is sufficient to run everything.. is that right?

Comment (crsven) [inline Cakefile]: With your changes, yes. I think `spec:server` was not correctly running before

**Pull request merged**